code/game/objects/items/dice.dm

```diff
@@ -71,6 +71,62 @@
 		result = roll(sides)
 	update_appearance()
 
+/obj/item/dice/attack_self(mob/user)
+	diceroll(user, TRUE)
```

Comment:
```suggestion
	diceroll(user, in_hand = TRUE)
```

`TRUE` as an un-named arg is not intuitive here, so let's make it as explicit as possible to be nice to future people who need to look at this code.